packages/engines/src/index.ts

```diff
@@ -62,11 +62,19 @@ path.join(__dirname, '../query-engine-darwin-arm64')
 path.join(__dirname, '../query-engine-debian-openssl-1.0.x')
 path.join(__dirname, '../query-engine-debian-openssl-1.1.x')
 path.join(__dirname, '../query-engine-debian-openssl-3.0.x')
+path.join(__dirname, '../query-engine-linux-arm64-openssl-1.0.x')
```

Comment:
Do we also have these builds for the schema engine? If so, should we add them to the CLI annotations we have?

Reply:
I don't think annotations for schema engine need to be added here because it is not supposed to be deployed. But I'm also not familiar with the tooling that requires these annotations, perhaps @Jolg42 can answer better.

Reply:
Actually if my understanding is correct that only the files that need to be deployed on Vercel need to be annotated, then perhaps we need to, on the contrary, trim this list down instead of expanding it? (i.e. remove darwin and unnecessary linux builds)